src/content/docs/accounts/accounts/automated-user-management/tutorial-manage-users-groups-scim.mdx

```diff
@@ -0,0 +1,694 @@
+
+If you are using a third-party identity provider, configure it to use **Bearer Token authorization** and plug in your API access token. Refer to your identity provider's documentation for help configuring this. Once configured, you're all set to import users and groups!
+
+The SCIM 2.0 protocol requires user and group data to be sent in accordance with a few core _schemas_ and encoded in JSON. See [RFC 7643 - SCIM Core Resources and Extensions](https://datatracker.ietf.org/doc/html/rfc7643#section-4) and [RFC 7643 - JSON Representation](https://datatracker.ietf.org/doc/html/rfc7643#section-8) for the specifics. Refer to [RFC 7644 - SCIM Protocol](https://datatracker.ietf.org/doc/html/rfc7644) for more information about the protocol used in this
+tutorial.
+
+For all requests to the SCIM API, you must provide the bearer token in an `Authorization` header. Here's an example with `curl`:
+
+```
+curl -X 'GET' -H 'Accept: application/json' -H "Authorization: Bearer $TOKEN" 'https://scim-provisioning.service.newrelic.com/scim/v2/Users'
+```
+
+Any request in the rest of this tutorial will receive a **401 Unauthorized** response if the API access token is missing or invalid.
+
+Example response:
+
+<Collapser title="401 Unauthorized">
+  ```
+  {
+    "schemas": [
+      "urn:ietf:params:scim:api:messages:2.0:Error"
+    ],
+    "detail": "invalid authentication token",
+    "status": "401"
+  }
+  ```
+</Collapser>
+
+## Create users in your Authentication Domain [#create-users]
+
+If you are using a third-party identity provider, refer to their documentation for how to push users with SCIM.
+
+Otherwise, send a `POST` request to `/scim/v2/Users` to create a user. The following user attributes are **required**:
+* `userName` This identifier **must** be unique within your Authentication Domain. Use the user's email address. 
+* `emails` Email address of the user.
+* `active` Boolean indicating whether or not the user should be active or inactive within New Relic.
+
+It is recomended to provide the following attributes for the best user experience:
+* `name.givenName` The user's first or given name.
+* `name.familyName` The user's last or family name.
+* `timezone` The user's timezone in IANA Time Zone database format.
+
+Take note of the returned user `id`. To update a user in the future you will need to supply the same ID with the request.
+
+```
+curl -X 'POST' -H 'Content-Type: application/json' -H "Authorization: Bearer $TOKEN" 'https://scim-provisioning.service.newrelic.com/scim/v2/Users' --data-binary @- <<EOF
+{
+  "schemas": ["urn:ietf:params:scim:schemas:core:2.0:User"],
+  "userName": "bjensen@example.com",
+  "name": {
+    "familyName": "Jensen",
+    "givenName": "Barbara"
+  },
+  "emails": [
+    {
+      "primary": true,
+      "value": "bjensen@example.com"
+    }
+  ],
+  "active": true,
+  "timezone": "America/Los_Angeles"
+}
+EOF
+```
+
+Example responses:
```

Comment:
This is a nit, so take it or leave it. This feels like it should be:

```suggestion
### Example responses
```

Regular paragraph text can get lost between long example snippets. If it becomes a header though, the `:` should go. But don't go all "Capitals Paragraph Case" though unless it's a proper noun.

Again, this is just an editor nit and I really hope that you'll ignore it if it doesn't match your style.